app/model/commands/UpdateAtomCommand.scala

```diff
@@ -36,9 +36,18 @@ case class UpdateAtomCommand(id: String, atom: MediaAtom, override val stores: D
     log.info(s"Update atom changes ${atom.id}: $diffString")
 
     val changeRecord = ChangeRecord.now(user)
+    val atomIsPublished = existingAtom.contentChangeDetails.published
```

Comment:
could this be a boolean to make the lines below a bit simpler?

Reply:
I've abstracted it out to a separate function